.eslintrc.js

```diff
@@ -14,5 +15,7 @@ module.exports = {
   rules: {
     "react/jsx-props-no-spreading": "warn",
     "react/require-default-props": "off",
+    "react/function-component-definition": "off",
+    "no-restricted-exports": ["off", { restrictedNamedExports: ["default"] }],
```

Comment:
Same as with function-component-definition rule. 

In general, preferring named exports is a good rule. We might want to shift to it.